source/boards/working-with-boards.rst

```diff
@@ -1,18 +1,55 @@
 Working with Boards
 ===================
 
-Adding new Boards
------------------
+Using Boards begins with selecting the type of board you want to use. The containers within a board, called cards, are used for tasks. Views are a way to group cards based on their property (e.g., priority, status, etc).
```

Comment:
Recommend not introducing a new concept of "containers" here. Maybe: "A board contains cards, which typically track tasks or topics, and views, which define how to display the cards, or a subset of them. Views can display cards in a board, table, or gallery layout, optionally filtered and grouped by a property (e.g., priority, status, etc)."